library/src/main/res/layout/dragonfly_lens_feedback_view.xml

```diff
@@ -21,13 +22,12 @@
                tools:tint="@android:color/white"
                tools:visibility="visible"/>
 
-    <TextView android:id="@+id/dragonflyLensLabelView"
-              android:layout_width="wrap_content"
-              android:layout_height="wrap_content"
-              style="@style/Dragonfly.Lens.Label"
-              android:background="@drawable/dragonfly_label_background"
-              android:layout_gravity="center"
-              android:visibility="gone"
-              tools:text="Hammer 25%"
-              tools:visibility="visible"/>
+    <LinearLayout
+        android:id="@+id/dragonflyLensLabelsContainer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
```

Comment:
Shouldn't we use dimens for this, so the user of the library is able to change the values as needed?